### PR TITLE
Wizard fix

### PR DESCRIPTION
**Pull request by cehsu**
- Deduplicates update-non-dependent id
- Enables multiple alternates

### CODE REVIEW

**Comment by cehsu** [comment]
The original test failure that prompted this is no longer a problem, but these changes still make sense.

File: src/js/edu-benefits/education-wizard.js [hunk 1]
```diff
@@ -41,12 +41,14 @@ function reInitWidget() {
 
   content.addEventListener('change', (e) => {
     const radio = e.target;
-    const otherChoice = radio.dataset.alternate;
-    const otherNextQuestion = container.querySelector(`#${otherChoice}`).dataset.nextQuestion;
-    if (otherNextQuestion) {
-      const otherNextQuestionElement = container.querySelector(`[data-question=${otherNextQuestion}]`);
-      closeStateAndCheckChild(otherNextQuestionElement, container);
-    }
+    const otherChoices = radio.dataset.alternate.split(' ');
```
Comment (jbalboni) [inline src/js/edu-benefits/education-wizard.js]: Why do this instead of making the name something like `update-non-dependent-transfer`?

Comment (cehsu) [inline src/js/edu-benefits/education-wizard.js]: My understanding is that benefits are transferred from sponsors to dependents, but if that is mistaken, `update-non-dependent-transfer` would be clearer. Perhaps @goldenmeanie can advise, unless you're familiar with benefits be transferred to non-dependents.

Comment (goldenmeanie) [inline src/js/edu-benefits/education-wizard.js]: I believe this is governing this part of the wizard

transfered-or-not

First option is not transferred (not a dependent)
Second option is transferred (a dependent/spouse)
Third option is not transferred (a dependent)

Third option is not considered transferred even though it is using a sponsor’s eligibility. The sponsor was not available to officially transfer the benefit. Does this help?

Comment (cehsu) [inline src/js/edu-benefits/education-wizard.js]: This is very helpful, thanks @goldenmeanie, @jbalboni is the current name ok given the provided explanation?

Comment (jbalboni) [inline src/js/edu-benefits/education-wizard.js]: My main concern there was just if we could come up with a name that didn’t require you to add the code to split dataset.alternate and loop through each item. If that doesn't make sense or isn't feasible, this looks fine to me.

**Review by jbalboni — APPROVED** [review]
Had one comment to see if we can remove some of the code that's added, but otherwise looks fine.

File: src/js/edu-benefits/education-wizard.js (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/education-wizard.js]: I was misunderstanding how `data-alternate` was being used; this change makes sense.

**Pull request merged**